servers/Azure.Mcp.Server/TROUBLESHOOTING.md

```diff
@@ -881,10 +881,25 @@ To export telemetry to Azure Monitor, set the `APPLICATIONINSIGHTS_CONNECTION_ST
 
 ### Development in VS Code
 
+#### Running Azure MCP Server Locally for Development
+
+When developing or debugging the Azure MCP Server locally, you can use `dotnet run` which automatically uses the configuration in `Properties/launchSettings.json`:
+
+```bash
+dotnet run --project servers/Azure.Mcp.Server/src/
+```
+
+The `launchSettings.json` file automatically configures:
+- **Command line arguments:** `server start --run-as-remote-http-service --outgoing-auth-strategy UseHostingEnvironmentIdentity`
+- **Environment variables** for Azure AD authentication and ASP.NET Core settings
```

Comment:
Always refer to it as "Entra" or "Entra ID", not "Azure AD". Microsoft rebranded the product some years ago, but the configurations hold the outdated name.